README.md

```diff
@@ -205,35 +222,36 @@ through to completion: authors should not expect that other project
 developers will take on responsibility for implementing their accepted
 feature.
 
-Modifications to active RFC's can be done in followup PR's.  We strive
+Modifications to active RFC's can be done in follow-up PR's. We strive
 to write each RFC in a manner that it will reflect the final design of
 the feature; but the nature of the process means that we cannot expect
 every merged RFC to actually reflect what the end result will be at
-the time of the next major release; therefore we try to keep each RFC
-document somewhat in sync with the language feature as planned,
-tracking such changes via followup pull requests to the document.
+the time of the next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes should
+be new RFCs, with a note added to the original RFC. Exactly what counts as a
+"very minor change" is up to the sub-team to decide. There are some more
+specific guidelines in the sub-team RFC guidelines for the [language](lang_changes.md),
+[libraries](libs_changes.md), and [compiler](compiler_changes.md).
 
-An RFC that makes it through the entire process to implementation is
-considered 'complete' and is moved to the 'complete' folder; an RFC
-that fails after becoming active is 'inactive' and moves to the
-'inactive' folder.
 
 ## Reviewing RFC's
 [Reviewing RFC's]: #reviewing-rfcs
 
 While the RFC PR is up, the shepherd may schedule meetings with the
 author and/or relevant stakeholders to discuss the issues in greater
-detail, and in some cases the topic may be discussed at the larger
-[weekly meeting]. In either case a summary from the meeting will be
+detail, and in some cases the topic may be discussed at a sub-team
+meeting. In either case a summary from the meeting will be
 posted back to the RFC pull request.
 
-The core team makes final decisions about RFCs after the benefits and
-drawbacks are well understood. These decisions can be made at any
-time, but the core team will regularly issue decisions on at least a
-weekly basis. When a decision is made, the RFC PR will either be
-merged or closed, in either case with a comment describing the
-rationale for the decision. The comment should largely be a summary of
-discussion already on the comment thread.
+A sub-team makes final decisions about RFCs after the benefits and drawbacks are
+well understood. These decisions can be made at any time, but the sub-team will
+regularly issue decisions. When a decision is made, the RFC PR will either be
+merged or closed, in either case with a comment describing the rationale for the
```

Comment:
On the libs team at least, we haven't been great at re-iterating the rationale in a separate comment, usually because we *have* been good about commenting on thread. Not sure if we want to keep this requirement.